src/core/caniuse.js

```diff
@@ -82,20 +82,23 @@ export async function run(conf) {
       return hyperHTML`<a href="${featureURL}">caniuse.com</a>`;
     }
   })();
+  const stats = hyperHTML`<dd class="caniuse-stats">
+    Fetching data from caniuse.com...
+  </dd>`;
   const definitionPair = hyperHTML`
     <dt class="caniuse-title">Browser support:</dt>
-    <dd class="caniuse-stats">${{
-      any: contentPromise,
-      placeholder: "Fetching data from caniuse.com...",
-    }}</dd>`;
+    ${stats}`;
   headDlElem.append(...definitionPair.childNodes);
-  await contentPromise;
+  const result = await contentPromise;
+  stats.textContent = "";
```

Comment:
Should we create an `emptyElement(el)` util? There are evidences of innerHTML being slow (so I suspect same from `textContent = ''`, https://stackoverflow.com/a/3955238) (not sure how slow in our case though)

Reply:
I don't think the performance here would matter as it would just be a few microseconds. `fillElement(el, content)` would be useful though, as it removes the need to create a separate variable.